macros/core/PGbasicmacros.pl

```diff
@@ -2889,7 +2889,7 @@ sub image {
 			my $altattrib = '';
 			if (defined $alt_list[0]) { $altattrib = 'alt="' . encode_pg_and_html(shift @alt_list) . '"' }
 			$out =
-				qq!<IMG SRC="$imageURL" class="image-view-elt $valign" tabindex="0" role="button"$width_attrib$height_attrib $out_options{extra_html_tags} $altattrib>!;
+				qq!<IMG SRC="$imageURL" class="image-view-elt $valign" style="max-width:100%;" tabindex="0" role="button"$width_attrib$height_attrib $out_options{extra_html_tags} $altattrib>!;
```

Comment:
Why use inline style for this?  There is a class `image-view-elt` that this could be applied to in the `imageview.scss` file.

Reply:
I commented above why I leaned inline. The motivating example has an inline width specification already, in the \ HTML for the table. And the purpose of this change is to play well with that inline specification.

If this styling were maintained in `imageview.scss`, what would happen if the HTML for the motivating problem ended up somewhere without `imageview.css`? I'm thinking of a third party application like PreTeXt, if it did not want to carry along the baggage of WeBWorK/PG styling files.

It doesn't matter for PTX for now. So if you don't think that reasoning outweighs avoiding inline styling, I will change it.

Reply:
I don't see why an inline style is needed because the other style is inline.  In general, inline styles should be avoided if at all possible.

If a system like PreTeXt does not want to use the intended rendering pipeline, it is that systems task to deal with handling style issues that arise as a result of doing so.  Ideally, there would only be one rendering system, and all that use PG problems would use that.  The point is to get what we render to work for everyone (with some options if needed).